lib/client.rb

```diff
@@ -85,10 +85,11 @@ def api_request(method, path, response_type = :json, **opts)
     # Creates a new request for the API call
     # with default and required values
     def new_request(method, path, response_type, **opts)
-      response_type = :json unless %i[json raw].include?(response_type)
+      response_type = :json unless %i[json raw xml].include?(response_type)
```

Comment:
Please update the documentation for `api_request` a few line before:
```
    # +response_type+:: +:json+ or +:raw+
```

It should be:
```
    # +response_type+:: +:json+, +:xml+ or +:raw+
```

Or something like that.

Reply:
All set https://github.com/yitsushi/jenkins2-api/pull/13/files#diff-6ba6b08815610e4ac61d82930beb2a36R67